CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.58.0
+
+- Action `fix_spec_file` can change a spec file - Packit now preserves that change. (#1679)
+- Packit now uses (py)rpkg instead of rebasehelper to parse `sources` files in dist-git repos and to interact with lookaside cache. (#1680)
+- `prepare-sources` command now has the `--create-symlinks/--no-create-symlinks` option, which enables copying the archive instead of symlinking. (#1682)
```

Comment:
I find the wording a bit confusing, maybe something like this would be better (if I get it correctly)?

`prepare-sources` command now has `--no-create-symlinks` option (together with the opposite `--create-symlinks`), which enables copying the archive instead of symlinking